myapp/app/views/tasks/index.html.erb

```diff
@@ -16,6 +17,7 @@
       <tr>
         <td><%= task.title %></td>
         <td><%= task.description %></td>
+        <td><%= task.created_at.strftime("%F %T") %></td>
```

Comment:
[Note] It's okay to use ("%F %T") here, but I think it is better to use the full format ("%Y-%m-%d %H:%M:%S") because it's easier to understand.

Reply:
I changed it to the full format ("%Y-%m-%d %H:%M:%S").